nautobot/dcim/filters.py

```diff
@@ -1039,6 +1041,11 @@ class DeviceFilterSet(
         field_name="virtual_chassis",
         label="Is a virtual chassis member",
     )
+    device_redundancy_group = NaturalKeyOrPKMultipleChoiceFilter(
```

Comment:
Do we want a `has_device_redundancy_group` filter?

Reply:
Does that hold up the merge?

Reply:
no